src/formatters/ToggleGroupFormatter.tsx

```diff
@@ -20,12 +21,12 @@ export function ToggleGroupFormatter<R, SR>({
   return (
     <span
       ref={cellRef}
-      className="rdg-group-cell-content"
+      className={groupCellContentClassname}
       tabIndex={-1}
       onKeyDown={handleKeyDown}
     >
       {groupKey}
-      <svg viewBox="0 0 14 8" width="14" height="8" className="rdg-caret">
+      <svg viewBox="0 0 14 8" width="14" height="8" className={caretClassname}>
```

Comment:
I think where possible, we can move the styles in the file that use them. This way we can encapsulate styles and I think that is what CSS-in-JS libraries promote. Wdyt?